test/suites/integration/bsp/submit-proofs.test.ts

```diff
@@ -168,28 +169,31 @@ describeBspNet(
       );
     });
 
-    it(
-      "BSP stops storing last file",
-      { skip: "Not implemented yet. Needs RPC method to build proofs." },
-      async () => {
-        // TODO: Build inclusion forest proof for file.
-        // TODO: BSP-Three sends transaction to stop storing the only file it has.
-        console.log(fileMetadata);
-        // // Build transaction for BSP-Three to stop storing the only file it has.
-        // const call = bspThreeApi.sealBlock(
-        //   bspThreeApi.tx.fileSystem.bspStopStoring(
-        //     fileMetadata.fileKey,
-        //     fileMetadata.bucketId,
-        //     fileMetadata.location,
-        //     fileMetadata.owner,
-        //     fileMetadata.fingerprint,
-        //     .fileSize,
-        //     false
-        //   ),
-        //   bspThreeKey
-        // );
-      }
-    );
+    it("BSP stops storing last file", async () => {
```

Comment:
This test could benefit from better assertions. Currently it only checks that a blockchain event has been fired, but we don't actually know what the effects on the physical node are. 

I don't know how much of the delete feature has been impl but if relevant - consider adding checks to:
- Use an RPC call to verify the file can not longer be found
- Check that an expecting logging line is

Reply:
There is more tests that are being skipped that are executed after this one. I believe we should indeed re-activate them to make sure the full scenario works as expected. This could be in a new PR.